### ISSUE TITLE
add picker options for filter_data_ui

### ISSUE THREAD

**Issue by RKonstantinR** [dreamRs/datamods#106]
Hi. Thanks for the great package. It would be nice to be able to pass options to pickerinput that are needed to replace the text on buttons (e.g. selectAllText or deselectAllText).

**Comment by pvictor** [comment]
Yes good idea, I'll look into it (need to do something general for all the widgets).

Victor
